src/app/zap-templates/zcl/data-model/chip/push-av-stream-transport-cluster.xml

```diff
@@ -0,0 +1,229 @@
+    <command code="0x00" source="client" name="AllocatePushTransport" optional="false" response="AllocatePushTransportResponse">
+      <description>This command SHALL allocate a transport and return a PushTransportConnectionID.</description>
+      <access op="invoke" privilege="manage"/>
+      <arg id="0" name="TransportOptions" type="TransportOptionsStruct"/>
+    </command>
+
+    <command code="0x01" source="server" name="AllocatePushTransportResponse" optional="false" disableDefaultResponse="true">
+      <description>This command SHALL be generated in response to an AllocatePushTransport command.</description>
+      <arg id="0" name="ConnectionID" type="int16u"/>
+      <arg id="1" name="TransportOptions" type="TransportOptionsStruct"/>
+      <arg id="2" name="TransportStatus" type="TransportStatusEnum" min="0x00" max="0x01"/>
+    </command>
+
+    <command code="0x02" source="client" name="DeallocatePushTransport" optional="false">
+      <description>This command SHALL be generated to request the Node deallocates the specified transport.</description>
+      <access op="invoke" privilege="manage"/>
+      <arg id="0" name="ConnectionID" type="int16u"/>
+    </command>
+
+    <command code="0x03" source="client" name="ModifyPushTransport" optional="false">
+      <description>This command is used to request the Node modifies the configuration of the specified push transport.</description>
+      <access op="invoke" privilege="manage"/>
+      <arg id="0" name="ConnectionID" type="int16u"/>
+      <arg id="1" name="TransportOptions" type="TransportOptionsStruct"/>
+    </command>
+
+    <command code="0x04" source="client" name="SetTransportStatus" optional="false">
+      <description>This command SHALL be generated to request the Node modifies the Transport Status of the transport.</description>
+      <access op="invoke" privilege="manage"/>
+      <arg id="0" name="ConnectionID" type="int16u" isNullable="true"/>
+      <arg id="1" name="TransportStatus" type="TransportStatusEnum" min="0x00" max="0x01"/>
+    </command>
+
+    <command code="0x05" source="client" name="ManuallyTriggerTransport" optional="false">
+      <description>This command SHALL be generated to request the Node to manually start the specified push transport.</description>
+      <arg id="0" name="ConnectionID" type="int16u"/>
+      <arg id="1" name="ActivationReason" type="TriggerActivationReasonEnum" isNullable="true" min="0x00" max="0x02"/>
+      <arg id="2" name="TimeControl" type="TransportMotionTriggerTimeControlStruct" optional="true" isNullable="true"/>
+    </command>
+
+    <command code="0x06" source="client" name="FindTransport" optional="false" response="FindTransportResponse">
+      <description>This command SHALL return the Stream Options Configuration for the specified push transport.</description>
+      <arg id="0" name="ConnectionID" type="int16u" optional="true" isNullable="true"/>
```

Comment:
Spec does not describe what happens when this field is omitted.  Spec issue needed.

Why are all these things both optional and nullable?  That usually indicates someone was confused about what optional and nullable mean....